websockets/http.py

```diff
@@ -34,20 +53,38 @@ def read_request(stream):
     ``stream`` is an :class:`~asyncio.StreamReader`.
 
     Return ``(path, headers)`` where ``path`` is a :class:`str` and
-    ``headers`` is a :class:`~email.message.Message`. ``path`` isn't
-    URL-decoded.
+    ``headers`` is a list of ``(name, value)`` tuples.
```

Comment:
I could, but I like using basic Python types in low level APIs when possible. A list of pairs is pretty manageable.